90zfsbootmenu/zfsbootmenu-help.sh

```diff
@@ -0,0 +1,195 @@
+
+$( colorize lightblue "[ALT+D] set bootfs" )
+Set the selected boot environment as the default for the pool.
+
+The operation will gracefully fail if the pool can not be set $( colorize red "read/write" ).
+
+$( colorize lightblue "[ALT+S] snapshots" )
+Access a list of snapshots of the selected boot environment. New boot environments can be created here.
+
+$( colorize lightblue "[ALT+C] cmdline" )
+Temporarily edit the kernel command line that will be used to boot the next kernel and boot environment. This change is not persisted between boots.
+
+$( colorize lightblue "[ALT+P] Pool status" )
+View the health and status of each imported pool.
+EOF
+SECTIONS+=("MAIN Main Menu")
+
+# shellcheck disable=SC2034
+read -r -d '' SNAPSHOT <<EOF
+$( colorize magenta "$( center "Snapshot Management")" )
+$( colorize lightblue "[ENTER] duplicate" )
+Creation method: $( colorize red "zfs send | zfs recv" )
+
+This creates a boot environment that does not depend on any other snapshots, allowing it to be destroyed at will. The new boot environment will immediately consume space on the pool equal to the $( colorize lightgray "REFER" ) value of the snapshot.
+
+A duplicated boot environment is commonly used if you need a new boot environment without any associated snapshots.
+
+The operation will gracefully fail if the pool can not be set $( colorize red "read/write" ).
+
+If $( colorize red "mbuffer" ) is available, it is used to provide feedback.
+
+$( colorize lightblue "[ALT+X] clone and promote" )
+Creation method: $( colorize red "zfs clone" ) , $( colorize red "zfs promote" )
+
+This creates a boot environment that is not dependent on the origin snapshot, allowing you to destroy the file system that the clone was created from. A cloned and promoted boot environment is commonly used when you've done an upgrade but want to preserve historical snapshots.
+
+The operation will gracefully fail if the pool can not be set $( colorize red "read/write" ).
+
+$( colorize lightblue "[ALT+C] clone" )
+Creation method: $( colorize red "zfs clone" )
+
+This creates a boot environment from a snapshot with out modifying snapshot inheritence. A cloned boot environment is commonly used if you need to boot a previous system state for a short time and then discard the environment.
+
+The operation will gracefully fail if the pool can not be set $( colorize red "read/write" ).
+
+$( colorize lightblue "[ALT+D] diff" )
+Compare the differences between the selected snapshot and the current state of the boot environment.
+
+The operation will gracefully fail if the pool can not be set $( colorize red "read/write" ).
+EOF
+SECTIONS+=("SNAPSHOT Snapshot Management")
+
+# shellcheck disable=SC2034
+read -r -d '' KERNEL <<EOF
+$( colorize magenta "$( center "Kernel Management")" )
+$( colorize lightblue "[ENTER] boot" )
+Immediately boot the selected kernel in the boot environment, with the kernel command line shown at the top of the screen.
```

Comment:
"Boot into the boot environment with the kernel selected kernel immediately" ? Or close to this, I'm not 100%

Reply:
Maybe "Immediately boot the chosen kernel in the selected boot environment"